gnucash/report/trep-engine.scm

```diff
@@ -1032,7 +1032,21 @@ be excluded from periodic reporting.")
       gnc:pagename-display (N_ "Sign Reverses")
       "m1" (G_ "Reverse amount display for certain account types.")
       'global
-      (keylist->vectorlist sign-reverse-list))))
+      (keylist->vectorlist sign-reverse-list)))
+
+    (gnc:register-trep-option
+     (gnc:make-multichoice-callback-option
+      gnc:pagename-display (N_ "Running Balance")
+      "n" (G_ "Display a running balance?")
+      running-balance
+      (list
+       (vector 'none   (G_ "None"))
+       (vector 'account (G_ "Retrieve from Underlying Accounts"))
+       (vector 'report (G_ "Calculate as per Report Sort Order")))
+      #f
+      (lambda (x)
+        (set! running-balance x)
+        (apply-selectable-by-name-display-options)))))
```

Comment:
A `gnc:make-multichoice-callback-option` is only needed if the option _must_ e.g. toggle other options' sensitivities. A `gnc:make-multichoice-option` is more than sufficient here. There's no need for the `running-balance` variable being kept and modified.

Reply:
Thanks for the pointer, that makes sense. I "borrowed" the code from the "Hide/Single column/Two column" above but you are right that option does more than mine. Will convert to the simpler gnc:make-multichoice-option